src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/util/EntityParser.java

```diff
@@ -11,6 +12,7 @@
     private static final ObjectMapper mapper = new ObjectMapper();
 
     public static Envelope parseEnvelopeMetadata(InputStream metadataStream) throws IOException {
+        mapper.enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
```

Comment:
Enabling features should be done globally. I believe with naming strategy you will solve both cases. So no need for feature.

Reply:
See above, but I don't see a problem with this be enabled globally after the mapping of Status is refactored, but that is not a task related to this ticket, so it should be dealt with later